lib/server/api/validators/timer.js

```diff
@@ -9,13 +9,22 @@ function typeOf(obj) {
   return type_exp.exec( Object.prototype.toString.call(obj) )[1];
 }
 
+const MAX_TIMEOUT_VALUE = Math.pow(2, 31) - 1;
```

Comment:
This is a pretty well established value doing the calculation isn't really helpful here

```suggestion
// 2^32 - 1
const MAX_TIMEOUT_VALUE = 2147483647;
```